### PR TITLE
refactor: move sharing section into a separate file

### PR DESCRIPTION
**Pull request by Floppy**
This is a proposal for splitting up the big old playbook.md. If this is acceptable, we can do the rest.

### CODE REVIEW

**Comment by johnwaterworth** [comment]
Note that we created a proposed structure for the Playbook last year:
https://docs.google.com/document/d/1-KDsKQ3iHi1LPBwVX4NduC-9KSbsf9iRCzQqY7r6v-k/edit#heading=h.3779j478gxme

**Comment by erbridge** [comment]
@johnwaterworth It's almost definitely easier to split the large file into separate files _before_ we worry about reorganizing the content.

**Comment by Floppy** [comment]
It worked in local dev for me, but I could do with someone else proving that before we merge :)

**Comment by lawrence-forooghian** [comment]
How would I test this locally?

File: playbook.md [hunk 1]
```diff
@@ -2247,51 +2247,9 @@ If there is a book relevant to your work that you would like to read, make a req
 
 dxw will pay for the equipment needed to enable you to work comfortably and safely. dxw also offers a budget of £25 towards a work backpack. Use the `dxw-purchasing` Slack channel to request any equipment you need, or talk to anyone in business operations if you are unsure if a purchase will be covered.
 
-## Sharing
+[Sharing](sections/sharing.md ':include')
```
Comment (lawrence-forooghian) [inline playbook.md:2250]: I'm struggling to find any information about this `':include'` or what it does. Could you point me in the right direction?

**Comment by erbridge** [comment]
This runs fine locally, except that the files don't appear to load in a stable order. Repeated reloads cause them to jump around the page. The docs for `:include` can be found here https://docsify.js.org/#/embed-files. Looks like there's an open issue about this problem: https://github.com/docsifyjs/docsify/issues/970

**Comment by erbridge** [comment]
And an open PR to fix it: https://github.com/docsifyjs/docsify/pull/1102

**Comment by erbridge** [comment]
#365 Will make this possible.

**Comment by erbridge** [comment]
Closing as we'll need a different approach for the new architecture.

**Pull request closed without merging**